src/App.jsx

```diff
@@ -1,28 +1,32 @@
 import React, { useState, useEffect } from 'react';
 import { getAllData } from './util/index';
+import RecipeList from './components/RecipeList';
+import { recipes } from './list';
+
 
 const URL = 'http://localhost:8000/api/v1/';
 
 function App() {
-  
-  const [message, setMessage] = useState(''); 
 
-  useEffect(() => {
+  // const [message, setMessage] = useState('');
```

Comment:
Why is this code commented out? Is it for future use, or was it for local debugging? If it was for local debugging I would suggest deleting it, but if it is needed for debugging in the future you might want to leave it.